extensions/guacamole-auth-radius/src/main/resources/styles/radius.css

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+
+.radius-challenge-response-field-container {
+    height: 100%;
+    width: 100%;
+    position: fixed;
+    left: 0;
+    top: 0;
+    display: table;
+    background: white;
+}
+
+.radius-challenge-response-field {
+    width: 100%;
+    display: table-cell;
+    vertical-align: middle;
+}
+
+.radius-challenge-response-field input[type="submit"] {
+    display: none !important;
+}
+
+.radius-challenge-response-field.loading iframe {
```

Comment:
Is there an `iframe` anywhere?

Orrrrr ... is this just copypasta from the guacamole-auth-duo CSS?